zipline-kotlin-plugin/build.gradle.kts

```
@@ -17,6 +17,14 @@ dependencies {
   compileOnly(libs.auto.service.annotations)
 }
 
+kotlin {
+  sourceSets {
+    all {
```

I believe `all` eagerly configures (not that it really matters much here)
```suggestion
    configureEach {
```